README.md

```diff
@@ -1,76 +1,61 @@
 # nvim-lspconfig
 
-Collection of common configurations for the [Nvim LSP client](https://neovim.io/doc/user/lsp.html).
-
-## Best-effort, unsupported
-
+Collection of common configurations for Neovim's built-in [language server client](https://neovim.io/doc/user/lsp.html).
 The configurations here are strictly **best effort and unsupported**.
 
-This repo is (1) a place for Nvim LSP users to collaboratively provide starting
-examples for the many LSP backends out there, and (2) a reference for the
-current best practice (or most popular) regarding choice of server, setup, etc.
-
-## Requires Nvim 0.5 HEAD
-
-While Nvim LSP undergoes development, the configs in this repo assume that you
-are using the latest [Nvim HEAD/nightly build](https://github.com/neovim/neovim/releases/tag/nightly).
-
-Update Nvim and nvim-lspconfig before reporting an issue.
+This repo is :
 
-## Contributions
-
-It's up to you to send improvements so that these configs align with current
-best practices for a given language.
-
-1. Read [CONTRIBUTING.md](CONTRIBUTING.md).
-   Ask questions in [Neovim Gitter](https://gitter.im/neovim/neovim).
-2. Choose a language from [the coc.nvim wiki](https://github.com/neoclide/coc.nvim/wiki/Language-servers) or
-  [emacs-lsp](https://github.com/emacs-lsp/lsp-mode#supported-languages).
-3. Create a new file at `lua/lspconfig/SERVER_NAME.lua`.
-   - Copy an [existing config](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/)
-     to get started. Most configs are simple. For an extensive example see
-     [texlab.lua](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/texlab.lua).
+* a place for users of Neovim's language server client to collaboratively provide starting 
+  examples for the many LSP backends out there
+* a reference for the current best practice (or most popular) regarding 
+  choice of server, setup, etc.
```

Comment:
I'd recommend staying away from "best choice" recommendations; rather focus on documenting configuration options for servers and hints for getting them installed and running (where not trivial).

Reply:
I hadn't touched a lot of the original language, so this was good feedback. I've tweaked the places it occurs, but let me know if the changes didn't address the issue.